Sources/Implementation/DefaultDatafileHandler.swift

```diff
@@ -159,19 +159,6 @@ class DefaultDatafileHandler: OPTDatafileHandler {
         }
     }
     
-    @objc
```

Comment:
why was this removed?

Reply:
It is not used anywhere. probably leftover from previous changes.

Reply:
leftover from ios 9 support. got it. :)